src/components/shared/FormInfo.js

```diff
@@ -12,135 +13,139 @@ const StyledGroup = styled.div`
     }
 `;
 
-class FormInfo extends Component {
-    constructor(props) {
-        super(props);
+const FormInfo = (props) => {
+    let elemInputs = [];
+    let saveInputs = [];
+
+    const initialState = {
+        id: '',
+        title: null,
+        date: null,
+        url: null,
+        overview: null,
+        time: null
+    };
 
-        this.initialState = {
-            id: '',
-            title: null,
-            date: null,
-            url: null,
-            overview: null,
-            time: null
-        };
+    const [genreState] = useState(['Select Genre', 'Horror', 'Action', 'Comedy']);
+    const [valueState, setValue] = useState(genreState[0]);
+    const [dataForm, setData] = useState(initialState);
 
-        this.newState = null;
+    const handleChange = useCallback((e) => {
+        setValue(e.target.value);
+        }, []
+    );
 
-        this.elemInputs = [];
-        this.saveInputs = [];
+    const getInputs = () => {
+        if (props.showEditPage) {
+            elemInputs = document.querySelectorAll('.Edit input');
```

Comment:
Interacting with real DOM elements completely gets out of React philosophy!
You should use controlled forms instead! Please, read more: https://reactjs.org/docs/forms.html